pkg/networkservice/mechanisms/kernel/kernelvethpair/common.go

```diff
@@ -64,7 +64,9 @@ func appendInterfaceConfig(ctx context.Context, conn *networkservice.Connection,
 				},
 				Link: &linuxinterfaces.Interface_Veth{
 					Veth: &linuxinterfaces.VethLink{
-						PeerIfName: linuxName,
+						PeerIfName:           name + "-veth",
+						RxChecksumOffloading: linuxinterfaces.VethLink_CHKSM_OFFLOAD_DISABLED,
+						TxChecksumOffloading: linuxinterfaces.VethLink_CHKSM_OFFLOAD_DISABLED,
```

Comment:
Do we need to apply these changes for line 51?

Reply:
Good catch, done :)